spacy_llm/models/rest/openai/registry.py

```diff
@@ -8,6 +8,49 @@
 
 _DEFAULT_TEMPERATURE = 0.0
 
+
+@registry.llm_models("spacy.OpenAI.v")
```

Comment:
The suspense is killing me - what version would it be?

Reply:
Gotta keep you on your toes.